Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -30,7 +30,8 @@ struct FPTIBatchData: Codable {
     struct Event: Codable {
 
         /// UTC millisecond timestamp when a networking task started establishing a TCP connection. See [Apple's docs](https://developer.apple.com/documentation/foundation/urlsessiontasktransactionmetrics#3162615).
-        ///
+        /// The position of the button in the list of available payment methods, associated with the payments ready flow
+        let buttonRank: Int?
```

Comment:
Lets make sure to add some tests for these as well

Reply:
added here! 5c4173c6bfb6a5d0ff5e455eea98f4da0ef34525